packages/lit-dev-content/site/css/articles.css

```diff
@@ -0,0 +1,298 @@
+main {
+  /* Colors */
+  --docs-h1-underline-color: var(--color-blue);
```

Comment:
Seems like a lot is copy-pasted from docs.css.

It would also be nice to factor out the common styles.

Also, is everything here actually needed? E.g. I don't see use of the next/previous buttons in the example article, but the CSS is copied  here.

Reply:
Next and previous buttons are there but hidden. docs.css is actually loaded. articles.css is just some stuff on top of docs.css.

Though it was meant to start out as its own thing there is some cruft. Let me take another pass at it